server/core/src/main/java/dev/slimevr/tracking/trackers/Tracker.kt

```diff
@@ -56,7 +56,7 @@ class Tracker @JvmOverloads constructor(
 	 * Has solved position + rotation (Vive trackers)
 	 */
 	val isComputed: Boolean = false,
-	val imuType: IMUType? = null,
+	var imuType: IMUType? = null, // TrackersHID has to assign after the device is created
```

Comment:
but a tracker should be initialized when the tracker info is received, not when the device is created...

Reply:
i can wait for the info packet before creating the tracker, but that depends if the tracker has time to send the info packet, it must be awake but not moving

Reply:
this sounds like something that should be fixed on esb protocol.
idk, i would like more opinions if we should let trackers be lazily updated or for a tracker to be initialized it should contain the info for its initialization

Reply:
i can update it to wait for the info packet, but it means the tracker may not get picked up quickly
a user probably does not constantly move around right after a device is started, so it is ok
devices were allowed to override its data, like MCUType, only for trackers it cannot be overridden without modifying base tracker

Reply:
after some more use, it doesnt seem like it is much of an issue as i thought